README.md

```diff
@@ -1,108 +1,148 @@
 # About
 
-This is the official repository of the evaluation codec for the Adaptive Scalable Texture Compression (ASTC) standard.
+This is the official repository for the Arm® Adaptive Scalable Texture
+Compression (ASTC) Encoder, `astcenc`, a command-line tool for compressing
+and decompressing images using the ASTC texture compression standard.
 
-ASTC technology developed by ARM® and AMD has been adopted as an official extension to both the Open GL® and OpenGL ES graphics APIs. ASTC is a major step forward in terms of image quality, reducing memory bandwidth and thus energy use.
+## The ASTC format
 
-The **ASTC Evaluation Codec** is a command-line executable that compresses and decompresses images using this texture compression standard.
+The ASTC compressed data format, developed by Arm® and AMD, has been adopted as
+an official extension to the Open GL®, OpenGL ES, and Vulkan® graphics APIs. It
+provides a major step forward both in terms of image quality at a given
+bitrate, and in terms of the format and bitrate flexibility available to
+content creators. This allows more assets to use compression, often at a
+reduced bitrate compared to legacy formats, reducing memory bandwidth and
+energy consumption.
 
-Running on a standard PC, this tool allows content developers to evaluate the impressive quality and size benefits offered by this texture compression format, in advance of it being available in GPU  hardware.
+The ASTC format specification is available here:
 
-# License #
-By downloading the packages below you ackowledge that you accept the End User Licence Agreement for the ATSC Codec.
-See [license.txt](license.txt) file for details.
+* [ASTC Specification v1.0 (PDF)](./Documentation/ASTC%20Specification%20v1.0.pdf)
```

Comment:
I wonder if we should point to the Khronos Data Format specification instead...

There have been a few bug fixes and clarifications through the years and those won't have made their way to our legacy document.. (I also have unpublished changes to the Data Format specification for the low-precision decode modes.)

Reply:
Yup, makes sense. Will link to the full profile extension specification, and delete the old PDF. I may re-add a format overview markdown page in a later PR, simply because the Khronos plain text documents can be a little hard to read if all you want is an overview.